tools/release-scripts/release-1.5.sc

```diff
@@ -0,0 +1,177 @@
+import java.time.Instant
+
+import $ivy.`com.softwaremill.sttp::core:1.1.12`
+import $ivy.`com.typesafe.play::play-json:2.6.9`
+
+import $file.ui
+import $file.tokens
+import $file.changelog
+import $file.github
+import $file.jenkins
+import $file.context
+import context._
+import changelog.SemVer
+import ammonite.ops._
+import tokens.Credentials
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Promise}
+import ammonite.ops._
+
+type BranchName = String
+
+def checkIfShouldStartRelease(latestReleasedVersion: SemVer)(implicit buildContext: BuildContext) = {
+
+  implicit val path = buildContext.marathon.directory.path
+
+  val latestReleasedVersionString = latestReleasedVersion.toTagString()
+
+  val gitLogOutput = %%('git, 'log, s"$latestReleasedVersionString..releases/1.5").out.lines
```

Comment:
What is the motivation of using git instead of the Github API?

Reply:
What is the motivation of using Github API instead of git? :)